signingscript/docker.d/passwords.yml

```diff
@@ -294,6 +294,11 @@ in:
              {"$eval": "AUTOGRAPH_REFERENCE_BROWSER_PASSWORD"},
              ["autograph_apk"]
             ]
+          - ["https://autograph-external.stage.autograph.services.mozaws.net",
```

Comment:
Johan, in our chat you said to use new scopes. Is that necessary, or can I ride along with the existing `release-signing`?

Reply:
Mmmmm, that's a good point! Yeah, new scopes are unnecessary. We're just adding a new format. I agree, let's use `release-signing`. That'll be consistent with Desktop and all the formats we have there.